src/com/company/RootHandler.java

```diff
@@ -0,0 +1,54 @@
+package com.company;
+
+import com.sun.net.httpserver.HttpExchange;
+
+import java.io.IOException;
+import java.io.OutputStream;
+
+public class RootHandler extends HelloWorldHandler {
+
+    public RootHandler(NameRepository names) {
+        super(names);
+    }
+
+    @Override
+    public void handle(HttpExchange request) throws IOException {
+        switch(request.getRequestMethod()){
+            case "GET":
+                getHandler(request);
+            default:
+                notFound(request);
+        }
+        request.close();
+
+    }
+
+    private void getHandler(HttpExchange request) throws IOException {
+        var dateTime = new CurrentDateTime();
+        String response = "Hello Bianca" + filterNames() + " - the time on the server is "  + dateTime.getCurrentTime() + " on " + dateTime.getCurrentDate();
```

Comment:
So an interesting design choice here. You've hardcoded Bianca in this string. And that sort of makes sense because the requirements say that your name should always be in there. But should you hardcode here, or should your setup code for your name repository just add `bianca` when it's initialized? Can you think of reasons why this might be a little nicer than hardcoding this string here?

Reply:
Will work on this now!